extern/warprnnt/__init__.py

```diff
@@ -0,0 +1,103 @@
+"""
+Provides a RETURNN wrapper around `warp-transducer`:
+  https://github.com/HawkAaron/warp-transducer.git
```

Comment:
Remove the `.git`.

Reply:
Maybe you can also extend this a bit, by giving also adding refs/links to other RNN-T implementations, just for reference? Just list the other links I gave you. Of course make clear which of these is actually wrapped here.

Reply:
both comments are addressed in the latest commit.